compiler/z/codegen/OMRTreeEvaluator.cpp

```diff
@@ -14116,6 +14116,195 @@ bool isSettingOp(uint8_t byteValue, TR::InstOpCode::Mnemonic SI_opcode)
       return false;
    }
 
+TR::Register *
+OMR::Z::TreeEvaluator::bitpermuteEvaluator(TR::Node *node, TR::CodeGenerator *cg)
+   {
+   TR::Node *value = node->getChild(0);
+   TR::Node *addr = node->getChild(1);
+   TR::Node *length = node->getChild(2);
+
+   auto valueReg = cg->evaluate(value);
+   auto addrReg = cg->evaluate(addr);
+
+   TR::Register *tmpReg = cg->allocateRegister(TR_GPR);
+
+   static const int8_t BIT_PERMUTE_CONSTANT_UNROLL_THRESHOLD = 4;
+
+   TR::Register *resultReg = cg->allocateRegister(TR_GPR);
+
+
```

Comment:
Can we remove the double space?

Reply:
76d2d7b66d3d36effbffe552348645558c890be2